chainer/functions/activation/sigmoid.py

```diff
@@ -76,7 +76,6 @@ def sigmoid(x, use_cudnn=True):
 
     .. admonition:: Example
 
-        It maps the input values into the range of :math:`[0, 1]`.
```

Comment:
I deleted this line because the output html was broken in my environment.
![example](https://cloud.githubusercontent.com/assets/1277089/23854064/fb37de16-0832-11e7-90f1-c432d48b9fbf.png)